src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+type Shapes = "triangle" | "circle" | "rectangle";
+type Colors = "red" | "green" | "blue";
+type colorOrNumber = Colors | number;
 
+export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea(): number;
 }
 
-export class Triangle {
+const isNotATriangle = (a: number, b: number, c: number): boolean => {
+  const temp = [a, b, c].sort((a, b) => a - b);
+
+  return temp[2] >= temp[0] + temp[1] ? true : false;
+};
+
+const lengthIsLessThanZero = (...params: colorOrNumber[]): boolean => {
+  return params.some((param) => {
+    if (typeof param === "string") {
+      return !param.length ? true : false;
```

Comment:
What is return ! operator??
Or > >= <= <
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators
```suggestion
      return !param.length;
```